core/modules/user/user.module

```diff
@@ -998,6 +1009,54 @@ function user_account_form_validate($form, &$form_state) {
   }
 }
 
+/**
+ * Validation callback for password constraints.
+ *
+ * @see user_account_form().
+ * @see user_pass_reset_form().
```

Comment:
`@see` references shouldn't have anything after the reference (i.e. no full-stop): https://api.backdropcms.org/doc-standards#see

Reply:
Oops, that's of course right.